dcpquery-api.yml

```diff
@@ -146,6 +146,12 @@ components:
         query:
           type: string
           description: Query to run, given as a SQL string.
+        params:
+          type: object
+          description: >
```

Comment:
Can you include an example that shows how this helps avoid the escaping issue

Reply:
Yes, absolutely, but I would like to merge https://github.com/HumanCellAtlas/dcp-cli/pull/359 first, and make a complete suite of documentation including the use of this.

Reply:
Filed https://github.com/HumanCellAtlas/query-service/issues/187 to track this.